src/views/ModelPlan/Table/index.tsx

```diff
@@ -51,10 +83,21 @@ const Table = ({ data, hiddenColumns, showDownloadButton }: TableProps) => {
         Header: t('requestsTable.headers.name'),
         accessor: 'modelName',
         Cell: ({ row, value }: any) => {
+          const filteredNameHistory: string[] = row.original.nameHistory?.filter(
+            (previousName: string) => previousName !== value
+          );
```

Comment:
I'm wondering if we could just slice the array at the the first index since we know that is the current model name instead of filtering?  I understand the possibility that a model name could return to the same name, but this may be still a wanted functionality to give an accurate history